hatchery/src/instance.rs

```diff
@@ -48,6 +51,7 @@ impl Instance {
             arg_buf_ofs,
             arg_buf_len,
             heap_base,
+            self_id_buf_ofs,
```

Comment:
this is not really a "buffer", `self_id_ofs` is more appropriate

Reply:
Makes sense, will change